internal/authz/repository/eventsourcing/eventstore/token_verifier.go

```diff
@@ -18,30 +18,26 @@ type TokenVerifierRepo struct {
 	View                 *view.View
 }
 
-func (repo *TokenVerifierRepo) VerifyAccessToken(ctx context.Context, tokenString, appName, appID string) (userID string, clientID string, agentID string, err error) {
-	clientID, err = repo.verifierClientID(ctx, appName, appID)
-	if err != nil {
-		return "", "", "", caos_errs.ThrowPermissionDenied(nil, "APP-ptTIF2", "invalid token")
-	}
+func (repo *TokenVerifierRepo) VerifyAccessToken(ctx context.Context, tokenString, clientID string) (userID string, agentID string, err error) {
```

Comment:
Is the todo one line below still a todo?

Reply:
as discussed... we will solve it in a different PR